.github/CODEOWNERS

```diff
@@ -422,6 +422,8 @@
 /core/tests/ @nithusha21
 /core/tests/protractor_utils @U8NWXD
 /core/tests/protractor_desktop @U8NWXD
+/core/tests/protractor @U8NWXD
+/extensions/**/protractor.js @U8NWXD
```

Comment:
I'm not sure whether `glob` will catch this, can you remove line 422 and check whether the lint test passes?

Reply:
The lint check did indeed fail complaining about using `**`. I can get away with using `*` instead, but I noticed that the CODEOWNERS file already contains `**`, for example:

https://github.com/oppia/oppia/blob/9d1780352e310b23b4f0148b552ab1bc1343c999/.github/CODEOWNERS#L528

Out of curiosity, what is the rationale behind when `**` is okay?

Reply:
Done